README.md

```diff
@@ -13,15 +13,15 @@ untrusted parties coordinated by consensus algorithms.
 Incubation Notice
 -----------------
 
-This project is a Hyperledger project in _Incubation_. It was proposed to the 
+This project is a _active_ Hyperledger project. It was proposed to the 
```

Comment:
Use "an" instead of "a" in "This project is **a** _active_ Hyperledger project. It was proposed to the "